docs/ossm/README.md

```diff
@@ -0,0 +1,9 @@
+[Return to Project Root](../../)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). For OpenShift Service Mesh 3, this documentation should be prioritized over the [Sail Operator documentation](../README.md). However, the official source of truth is the product documentation available at [docs.openshift.com](https://docs.openshift.com). If a particular topic is not covered here, you can refer to the [Sail Operator documentation](../README.md) or [Istio.io](https://istio.io) documentation as supplementary resources.
```

Comment:
If this is going to remain live and accessible here after it has been moved to prod docs, direct users to https://docs.redhat.com as docs.openshift is being decommissioned in Nov 2024.